src/main/java/com/linkedin/replica/jobs/models/Job.java

```diff
@@ -4,16 +4,17 @@
 
 
 
-    private  String jobID ;
-    private  String industryType;
-    private  String employmentType;
-    private  String jobFunctions;
-    private  String positionName;
-    private  String professionLevel;
-    private  String companyID;
-    private  String companyName;
-    private  String companyLocation;
-    private   String compnayPicture;
+    public  String jobID ;
```

Comment:
Private vars & set accessibility of attributes to true whenever you want to access them through reflection